app/models/elastic_search.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
+
+class ElasticSearch
+  def initialize(_index = nil)
```

Comment:
I'd like to have some method documentation.  Standard fare:  what does it do, what does it want, what does it output, and what will it do when things go bad.